web/src/components/xray/Xray.tsx

```diff
@@ -135,63 +170,41 @@ export const XrayRegistrationTask = (props: Props): ReactElement => {
                 <Tab
                   value="0"
                   sx={tabStyle}
-                  label={Config.xrayRegistrationTask.tab1Text}
+                  label={xrayConfig.tab1Text}
                   data-testid={"start-application-tab"}
                 />
                 <Tab
                   value="1"
                   sx={tabStyle}
-                  label={Config.xrayRegistrationTask.tab2Text}
+                  label={xrayConfig.tab2Text}
                   data-testid={"check-status-tab"}
                 />
               </TabList>
             </Box>
             <TabPanel value="0">
-              <div className="margin-top-3">
-                <UnlockedBy task={props.task} />
-                <Content>{props.task.summaryDescriptionMd || ""}</Content>
-                <Content>{getModifiedTaskContent(roadmap, props.task, "contentMd")}</Content>
-                <HorizontalLine />
-                <div className="h6-styling">{Config.xrayRegistrationTask.issuingAgency}</div>
-              </div>
-              <CtaContainer>
-                <ActionBarLayout>
-                  <div className="margin-top-2 mobile-lg:margin-top-0">
-                    <SecondaryButton
-                      isColor="primary"
-                      onClick={(): void => {
-                        setTabIndex(STATUS_TAB_INDEX);
-                      }}
-                      dataTestId="cta-secondary"
-                    >
-                      {Config.xrayRegistrationTask.callToActionSecondaryText}
-                    </SecondaryButton>
-                  </div>
-                  <PrimaryButton
-                    isColor="primary"
-                    onClick={(): void => {
-                      openInNewTab(callToActionLink);
-                    }}
-                    isRightMarginRemoved
-                    dataTestId="cta-primary"
-                  >
-                    {Config.xrayRegistrationTask.callToActionPrimaryText}
-                  </PrimaryButton>
-                </ActionBarLayout>
-              </CtaContainer>
+              <XrayTabZero
+                summaryDescriptionMd={xrayContent.summaryMd}
```

Comment:
Dust - for summaryDescriptionMd to ctaSecondaryText you could pass a single argument - "xrayContent"